test-suite-linker/src/main/scala/org/scalajs/bootstrap/TestSuiteLinker.scala

```diff
@@ -16,7 +19,7 @@ import org.scalajs.linker.irio._
 object QuickLinker {
   /** Link the Scala.js test suite on Node.js */
   @JSExport
-  def linkTestSuiteNode(irFilesAndJars: js.Array[String], outputPath: String): Unit = {
+  def linkTestSuiteNode(irFilesAndJars: js.Array[String], outputPath: String): js.Promise[Unit] = {
```

Comment:
Why is this change not accompanied by a change in `Build.scala` somewhere? It looks suspicious to me. Aren't we discarding that `Promise` without looking at its result (nor even waiting for it)?

Reply:
Done.